docs/user_guide.md

```diff
@@ -556,76 +518,174 @@ If you are using a self-signed cert, make sure to copy the CA cert into ```/etc/
 When an image is signed, it has a tick shown in UI; otherwise, a cross sign(X) is displayed instead.  
 ![browse project](img/content_trust.png)
 
-### Vulnerability scanning via Clair 
-**CAUTION: Clair is an optional component, please make sure you have already installed it in your Harbor instance before you go through this section.**
+## Vulnerability Scanning
 
-Static analysis of vulnerabilities is provided through open source project [Clair](https://github.com/coreos/clair). You can initiate scanning on a particular image, or on all images in Harbor. Additionally, you can also set a policy to scan all the images at a specified time everyday.
+Harbor provides static analysis of vulnerabilities in images through the open source [Clair](https://github.com/coreos/clair) project. You can also connect Harbor to additional vulnerability scanners, also known as interrogation services. The additional scanners that are currently supported are [Anchore](https://anchore.com/harbor/) and [Aqua](https://github.com/aquasecurity/harbor-scanner-aqua).
 
-**Vulnerability metadata** 
+You might need to connect Harbor to other scanners for corporate compliance reasons, or because your organization already uses a particular scanner. Different scanners also use different vulnerability databases, capture different CVE sets, and apply different severity thresholds. By connecting Harbor to more than one vulnerability scanner, you broaden the scope of your protection against vulnerabilities.
```

Comment:
@xaleeks I reworded this to "It might be necessary to connect Harbor to other scanners..." in https://github.com/goharbor/harbor/pull/10029/commits/0969773f4ab9f62366f561566f33d5409a0b5bd5.